{{cookiecutter.project_slug}}/clients/web/vue3/src/services/users/models.js

```diff
@@ -1,14 +1,29 @@
-import Model, { fields } from '@thinknimble/tn-models'
+import { z } from 'zod'
+import { readonly } from '@thinknimble/tn-models-fp'
+import { baseModelShape,  } from '../base-model'
 
-import UserAPI from './api'
+export const userShape = {
+  ...baseModelShape,
```

Comment:
Why are these all "shapes"?

Reply:
https://github.com/thinknimble/tn-models-fp/issues/145

Basically they comply with type from `zod` library. 

We could call them `models` if we wanted to, but they still would need to match the `ZodRawShape` structure.

Reply:
gotcha. So we call them shapes because zod calls them shapes. Fair enough.
If it's an internal only zod thing that most consumers of zod wouldn't know about I would consider calling it something else.
Models may or may not be the answer there.

Reply:
Yea I like the idea of shapes over models, but I could also be content with `structure` `entity` `interface` <--- maybe less this one. I think models is a bit incorrect since these aren't true db backed 'models'

Reply:
shapes seems fine then